disnake/automod.py

```diff
@@ -127,11 +135,21 @@ class AutoModBlockMessageAction(AutoModAction):
 
     _metadata: AutoModBlockMessageActionMetadata
 
-    def __init__(self) -> None:
+    def __init__(self, custom_message: Optional[str] = None) -> None:
         super().__init__(type=AutoModActionType.block_message)
 
+        self._metadata["custom_message"] = custom_message or ""
```

Comment:
This should only be set if it is provided, right now this always provides custom_message to the API but it is optional.
```suggestion
        if custom_message is not None:
            self._metadata["custom_message"] = custom_message
```